style.css

```diff
@@ -29,6 +34,81 @@ img {
     margin-bottom: 0.5em;
 }
 
+body svg {
+    box-sizing: initial;
+}
+
+/* navbar */
+
+nav {
+    background-color: var(--white)
+}
+
+nav svg {
+    font-size: inherit;
+    fill: var(--primary-gray);
+    width: 1em;
+}
+
+.search-input {
+    width: 50vw;
+}
+
+nav a {
+    text-decoration: none;
+    color: var(--primary-gray);
+}
+
+img {
+    margin: 0;
+    padding: 0;
+}
```

Comment:
I think we should adopt a new pattern, to keep our code organized. Let's move all the "generic" rules to the top of the file. And divide by some commonets. For example:

```
/* Generics */
```